src/core/include/openvino/op/scatter_nd_update.hpp

```diff
@@ -29,12 +29,12 @@ class OPENVINO_API ScatterNDUpdate : public util::ScatterNDBase {
     bool has_evaluate() const override;
 };
```

Comment:
@mitruska, do we still need opset headers etc.? May it is enough to introduce just new version of operation without opset?